### DISCUSSION TITLE
饼图如何实现 把鼠标放上去 某一块会变大的效果

### DISCUSSION THREAD

**Discussion by zhiyou1201** [antvis/G2#5727] [Ideas]
饼图如何实现 把鼠标放上去 某一块会变大的效果 像这样的
![image](https://github.com/antvis/G2/assets/75651528/d2d16bec-234e-4529-aaa6-90513c38de86)
![image](https://github.com/antvis/G2/assets/75651528/56402ab3-1e7d-4c13-9ecb-9cb42bdc9495)

https://github.com/antvis/G2/assets/75651528/1b291cab-beea-494e-bd6a-fdafb93e1788

**Comment by pearmini** [comment]
```js
chart.interval().state('active', { transform: 'scale(1.1, 1.1)' })
```

**Reply by zhiyou1201** [reply]
这个不管用

**Reply by pearmini** [reply]
要结合 elementHighlight 交互使用，看看文档中相关的部分。

**Reply by zhiyou1201** [reply]
文档上就没有这个啊  就是没看到文档上有相关的部分啊

**Reply by zhiyou1201** [reply]
还是帮忙写出来了  我实在找不到文档怎么写

**Reply by zhiyou1201** [reply]
我加上代码
.interaction('elementHighlight')
.state('active', { transform: 'scale(1.1, 1.1)' })
但是出现这样的了

https://github.com/antvis/G2/assets/75651528/176bbefb-84b6-402c-93c2-a88111a072fa

**Reply by zhiyou1201** [reply]
回复一下吧 谢谢了

**Reply by pearmini** [reply]
看了一下，这种场景目前还不支持

**Reply by zhiyou1201** [reply]
这是什么意思呢？  是我这个没办法实现这样的效果？

**Reply by zhiyou1201** [reply]
目前还不支持 我问的几个问题 都是不支持  哎   你们的文档就已经很不清楚了  功能还是很多不能实现  你们发展这么久了  怎么就这点水平 跟其他平台差太多了

**Reply by zhiyou1201** [reply]
你们更新文档在哪   我用的cdn 怎么升级  升级文档在哪 我想看看升级了什么

**Reply by zhiyou1201** [reply]
真是服了 什么玩意啊  重写了  换插件了   再也不用你们这个了  真是太差了  你们做的这叫什么啊  水平真有限   文档看着恶心 回复慢不说  还解决不了问题  功能还都实现不了  选择用你们这个  真是浪费时间

**Comment by zhiyou1201** [comment]
这个不管用啊

我的代码 
var chart = new G2.Chart({
					container: 'g2-zong-interval-cate',
					autoFit: true,
					paddingTop: 30
				});
				chart.coordinate({ type: 'theta', innerRadius: 0.4, outerRadius: 0.9 });
				chart.title({
					size: -28,
					title: '总利润',
					titleFontSize: 16,
					titleFontWeight: 'normal',
					titleFill: '#000000',
				});
				chart
					.interval()
					.data(this.zhuData.TotalProfit)
					.transform({ type: 'stackY' })
					.encode('y', 'value_text')
					.state('active', { transform: 'scale(1.1, 1.1)' })
					.encode('color', 'name')
					.label({
						text: (d) => d.value.toFixed(2),
						fontWeight: 'bold',
						offset: 14,
						transform: [
							{
								type: 'overlapDodgeY',
							}
						]
					})
					.label({
						text: 'name',
						position: 'spider',
						connectorDistance: 0,
						fontWeight: 'normal',
						fill: '#333333',
						fontSize: 14,
						fillOpacity: 1,
						textBaseline: 'bottom',
						transform: [
							{
								type: 'overlapDodgeY',
							}
						]
					})
					.axis({
						x: {
							title: '利润率',
							labelFontSize: 30,
							titlePosition: 'right'
						}
					})
					.tooltip({
						title: '总利润',
						items: [
							(d) => ({
								name: d.name,
								value: d.value.toFixed(2)
							})
						],
					})
					.style('radius', 4)
					.style('stroke', '#fff')
					.style('lineWidth', 2)
					.animate('enter', { type: 'waveIn' })
					.legend({
						color: {
							itemMarker: 'circle',
							itemLabelFontSize: 14,
							itemLabelFill: '#000000',
							layout: {
								justifyContent: 'center',
							}
						}
					})
					.scale('color', {
						range: ['#00c250', '#00c6c6', '#7299ff'],
					});
				chart
					.text()
					.style({
						text: '利润率',
						x: '50%',
						y: '54%',
						fontSize: 16,
						textAlign: 'center',
						// fill: '#02c1ff'
					})
					
				chart
					.text()
					.style({
						text: `${this.zhuData.lirun_lv.toFixed(2)}%`,
						x: '50%',
						y: '46%',
						fontSize: 18,
						textAlign: 'center',
						// fill: '#02c1ff'
					})

				chart.render();

**Comment by zhiyou1201** [comment]
回复一下吧

**Comment by zhiyou1201** [comment]
啊
